src/components/UI/Display/Cluster/ClusterLabels/DeleteLabelButton.tsx

```diff
@@ -1,10 +1,40 @@
-import React, { ComponentPropsWithRef, FC } from 'react';
-import Button from 'UI/Controls/Button';
+import { Anchor } from 'grommet';
+import React, { FC } from 'react';
+import styled from 'styled-components';
+import { Tooltip, TooltipContainer } from 'UI/Display/Tooltip';
 
-interface IButtonProps extends ComponentPropsWithRef<typeof Button> {}
+interface IButtonProps {}
+
+const StyledAnchor = styled(Anchor)`
```

Comment:
This is mostly taken from the styled Anchor element we use for deleting access control subjects